### DISCUSSION TITLE
v2/zh/guide/blog/category-and-tags

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#263] [Announcements]
# v2/zh/guide/blog/category-and-tags

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/blog/category-and-tags.html

**Comment by randolfly** [comment]
这样设计会不会分类和标签同质化很高了？可以分类增加嵌套结构吗？例如：
```js
- category
  - HTML
  - Web
```
会创建`/HTML.html`和`/HTML/Web.html`的有层级关系的索引结构，这样一方面和标签具有区分度，另一方面也符合文件系统的直观逻辑？

**Reply by randolfly** [reply]
当然作者的功能已经很好了，非常感谢！

**Comment by Charlieconomy** [comment]
仅有 blog 模式有次功能吗？docs 模式下如何使用？

**Comment by qq584686739** [comment]
博客+暗黑模式下，访问tags列表，tag的样式貌似有问题，字体和颜色和tag块颜色类似，导致字体看不清了~
